backend/x11/output.c

```diff
@@ -331,12 +338,179 @@ static void output_rollback_render(struct wlr_output *wlr_output) {
 	wlr_renderer_bind_buffer(x11->renderer, NULL);
 }
 
+static void update_x11_output_cursor(struct wlr_x11_output *output,
+		int32_t hotspot_x, int32_t hotspot_y) {
+	struct wlr_x11_backend *x11 = output->x11;
+
+	xcb_cursor_t cursor = x11->cursor; /* transparent cursor */
```

Comment:
Maybe we can rename `x11->cursor` to something like `x11->transparent_cursor` to avoid confusion?

Reply:
Sure thing!